Makefile

```diff
@@ -42,9 +49,10 @@ IELE_VM          := $(IELE_BIN)/iele-vm
 IELE_TEST_VM     := $(IELE_BIN)/iele-test-vm
 IELE_TEST_CLIENT := $(IELE_BIN)/iele-test-client
 
+SHELL=/bin/bash
```

Comment:
Why is this one?

Reply:
For some reason, if you don't set SHELL then setting PATH in a Makefile doesn't work on Mac. I only needed to do this so I could run the tests after building, so I can remove it if you prefer.

https://stackoverflow.com/a/25506676

Reply:
Include a link to this post in a comment above this then.